### PR TITLE
1-27

### PR DESCRIPTION
**Pull request by thibaultduponchelle**
Run also after Christmas since we have articles for 26 and 27

### CODE REVIEW

**Comment by thibaultduponchelle** [comment]
@openstrike @oalders

**Comment by thibaultduponchelle** [comment]
To fix the website until this is merged, I restarted the deploy job

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ on:
     branches:
       - 'main'
   schedule:
-    - cron: '0 0 1-25 12 *'  # December 1-25
+    - cron: '0 0 1-27 12 *'  # December 1-25
```
Comment (openstrike) [inline .github/workflows/build.yml:9]:
```suggestion
    - cron: '0 0 1-27 12 *'  # December 1-27 (usually 1-25)
```

**Review by openstrike — CHANGES_REQUESTED** [review]
Do we want to go beyond the 25th? Makes it less of an advent calendar and I don't recall that happening in previous years. If you keep the other 2 articles until next year that means we would only need another 23 - less work for everyone.

Anyway, the one change I would make is to keep the comment in sync with the data.

**Review by oalders — APPROVED** [review]

**Comment by oalders** [comment]
@openstrike I think we can consider them bonus articles for this year. Not every previous year has hit the same number of articles. 24 or 25 would be perfect but this seemed like a nice way to celebrate the calendar being back with a full slate of original content.

**Pull request merged**